Gemfile

```diff
@@ -55,6 +55,7 @@ gem 'particlerb'
 gem 'aasm', '4.4.0'
 gem 'closure_tree', '6.0.0'
 
+
```

Comment:
If you're not using the chartkick gem, you should probably revert the changes to the Gemfile and the Gemfile.lock file.